### PR TITLE
Improved Great People 5

### PR DESCRIPTION
**Pull request by etiennefd**
Highlights: 

- Special focus on China, India, Persia, Ethiopia, England

- Merged Persia and Iran 

- Great Spies
The following civs have received a fairly extensive Spy list (at least 6 names): India, Japan, Vikings, Spain, France, England, Germany, Russia, Poland, Italy, USA. 
The following civs have received 1-5 Spy names: China, Korea, Arabia, Portugal, Mughals, Turkey, Netherlands, Argentina, Mexico, Canada
The other civs have no Great Spies, either because I couldn't find any or because they're too unlikely to produce Great Spies in game (e.g. they're too early).

### CODE REVIEW

File: Assets/Python/GreatPeople.py [hunk 1]
```diff
@@ -275,40 +336,70 @@ def setup():
 		"Jamshetji Tata", # 19th
 		"Ardeshir Godrej", # 19th
 		iGlobal,
+		"V. O. Chidambaram Pillai", # 20th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Can you find a form of the name that isn't too long without initials?

File: Assets/Python/GreatPeople.py [hunk 2]
```diff
@@ -275,40 +336,70 @@ def setup():
...
 		"Chandragupta Maurya", # 4th BC
 		"Samudragupta", # 4th BC
 		iMedieval,
-		"Rajaraja Chola", # 12th
+		"Dhruva Dharavarsha", # 8th
+		"Mihira Bhoja", # 9th
+		"Rajaraja Chola", # 10th
 		iRenaissance,
 		"fRani Durgavati", # 16th
-		"Shivaji Bhosle", # 17th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Why did you remove him? The LH is Shahuji.

File: Assets/Python/GreatPeople.py [hunk 3]
```diff
@@ -486,7 +577,6 @@ def setup():
 		"Fotis Kafatos", # 20th
 	],
 	iGreatMerchant : [
-		"Kroisos", # 6th BC
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Why remove him?

File: Assets/Python/GreatPeople.py [hunk 4]
```diff
@@ -606,59 +696,146 @@ def setup():
 },
 iCivPersia : {
 	iGreatProphet : [
-		"Zoroaster", # 18-10th BC
+		"Mahabad", # legendary
+		"Zarathustra", # 18-10th BC
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:701]: 👍

File: Assets/Python/GreatPeople.py [hunk 5]
```diff
@@ -606,59 +696,146 @@ def setup():
...
 	],
 	iGreatScientist : [
+		"Ktesias", # 5th BC
 		"Ardashir", # 4th
+		"Borzuya", # 6th
+		"Paulos-e irani", # 6th
 		iMedieval,
-		"Al-Khwarizmi", # 9th
-		"Al-Razi", # 9th
+		"Muhammad ibn Musa al-Khwarizmi", # 9th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: If the "last name" is well known I think it's best to leave it at that.

File: Assets/Python/GreatPeople.py [hunk 6]
```diff
@@ -606,59 +696,146 @@ def setup():
...
+		iGlobal,
+		"Manny Mashouf", # 20th
+		"Nasser David Khalili", # 20th
+		"Omid Kordestani", # 20th
+		"Amir Ansari", # 20th
+		"Pierre Omidyar", # 20th
 	],
 	iGreatEngineer : [
-		"Artaxerxes", # 5th BC
+		u"Artaxsaça", # 4th BC
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: I would prefer the more phonetic transliteration "Artakhshathra" (see [here](http://www.jewishencyclopedia.com/articles/1827-artaxerxes-i))

File: Assets/Python/GreatPeople.py [hunk 7]
```diff
@@ -606,59 +696,146 @@ def setup():
...
+		iGlobal,
+		"Reza Shah Pahlavi", # 20th
+		"Mohammad Mossadegh", # 20th
 	],
 	iGreatGeneral : [
-		"Achaemenes", # 7th BC
-		"Xerxes", # 5th BC
-		"Mehrdad", # 2nd BC
+		"Haxamanis", # 7th BC
+		"Xsaya-rsa", # 5th BC
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Likewise, "Khashayarsha".

File: Assets/Python/GreatPeople.py [hunk 8]
```diff
@@ -1641,16 +1940,20 @@ def setup():
...
 		iGlobal,
+		"Thomas Edward Lawrence", # 20th
 		"fEmmeline Pankhurst", # 20th
 		"Clement Atlee", # 20th
+		"fDiana Spencer", # 20th
 	],
 	iGreatGeneral : [
 		"William the Conqueror", # 11th
 		"Richard the Lionheart", # 12th
+		"William Wallace", # 13th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: I don't think Scottish figures before unification should be included (especially those who fought against England)

File: Assets/Python/GreatPeople.py [hunk 9]
```diff
@@ -2487,6 +2868,14 @@ def setup():
 		iIndustrial,
 		"Henri Winkelman", # 20th
 	],
+	iGreatSpy : [
+		"fJohanna Brandt", # 19th
+		"Christiaan Snouck Hurgronje", # 19th
+		iGlobal,
+		"fMata Hari", # 20th
+		"Dirk Klop", # 20th
+		"François van 't Sant", # 20th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: That should be a unicode string.

**Review by dguenms — CHANGES_REQUESTED** [review]
Great additional names as always, only some small change requests.

File: Assets/Python/GreatPeople.py (hunk 1 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: Valliappan Olaganathan Chidambaram Pillai is a little long, isn't it? Apparently he's also known as Kappalottiya Tamizhan, "The Tamil Helmsman", so I'll put that.

File: Assets/Python/GreatPeople.py (hunk 2 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: Interesting. The XML tags say Shivaji, which is what I had checked, plus the fact that Shivaji seemed like a more important figure. The civilopedia also has a piece on Shivaji, not on Shahuji. The XML text however is Shahuji, and so is the constant in Consts.py. I'll put Shivaji back in the GP list, but what is the intention here?

File: Assets/Python/GreatPeople.py (hunk 5 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: I use Wikipedia titles as a heuristic to choose the best name to use, unless that name is not in the native language of the civ, but in this case I'm reverting it.

File: Assets/Python/GreatPeople.py (hunk 9 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: Good catch.

**Review by dguenms — APPROVED** [review]
